.github/workflows/quay.yml

```diff
@@ -29,30 +29,27 @@ jobs:
           VERSION=${TAG/v/}
           make all ORG=${ORG} VERSION=${VERSION}
 
-      - name: Add component to DCI (distributed-ci.io)
+      - name: Get repo name
+        run: echo "::set-output name=name::${GITHUB_REPOSITORY/*\/}"
+        id: repo
+
+      - name: Create DCI components
+        uses: tonyskapunk/dci-component@v0.2.0
```

Comment:
Nice!, maybe later we can have that in one of the GH DCI accounts.

Reply:
Sure, I can migrate it to an org